src/mococrw/padding_mode.h

```diff
@@ -46,7 +47,7 @@ class RSAPadding
      * @param key RSA public key that will be used for encryption
      * @return the maximum size of the data that can be encrypted in bytes.
      */
-    virtual int getDataMaxSize(const AsymmetricPublicKey& key) const = 0;
+    virtual int getDataMaxSize(AsymmetricPublicKey& key) const = 0;
```

Comment:
Why? This should not be a non-const. Otherwise, please use pointers. I don't want non-const refs in parameters because you don't see this when reading code.